tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -266,6 +281,26 @@ def _test_storage_type(storage_type, fsx_fs_id, region):
         assert_that(get_storage_type(fsx_fs_id, region)).is_equal_to("SSD")
 
 
+def get_deployment_type(fsx_fs_id, region):
+    logging.info("Getting DeploymentType from DescribeFilesystem API.")
+    fsx = boto3.client("fsx", region_name=region)
+    return (
+        fsx.describe_file_systems(FileSystemIds=[fsx_fs_id])
+        .get("FileSystems")[0]
+        .get("LustreConfiguration")
+        .get("DeploymentType")
+    )
+
+
+def _test_deployment_type(deployment_type, fsx_fs_id, region):
+    if deployment_type == "PERSISTENT_1":
+        assert_that(get_deployment_type(fsx_fs_id, region)).is_equal_to("PERSISTENT_1")
+    elif deployment_type == "SCRATCH_2":
+        assert_that(get_deployment_type(fsx_fs_id, region)).is_equal_to("SCRATCH_2")
+    else:
+        assert_that(get_deployment_type(fsx_fs_id, region)).is_equal_to("SCRATCH_1")
```

Comment:
minor: the code could be simplified as
```
if not deployment_type:
    deployment_type = "SCRATCH_1"
assert_that(get_deployment_type(fsx_fs_id, region)).is_equal_to(deployment_type)
```

Reply:
minor: it can  also be simplified as one sentence:
assert_that(get_deployment_type(fsx_fs_id, region)).is_equal_to(deployment_type if deployment_type else "SCRATCH_1")
Maybe we can also add a comment to explain that the default deployment_type for the cloudformation is "SCRATCH_1"

Reply:
Done!